### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by VitorGuedesMadeira**
### Capstone Project

- Build Home Page ✔️
- Build About Page ✔️
- Add Dinamically Cards ✔️
- Add Media Query (768px) ✔️
- Add mobile/desktop navigation menu ✔️
- Store data in array of objects (javascript file) ✔️ 

Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

Video: https://www.loom.com/share/1d15d1b5265e4d4bb97f007ea34b1813?t=2

download

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        <a class="toggle-menu" href="#">
+            <img class="toggle-img" src="./images/Header/mobile_menu.png" alt="">
+        </a>
+        <nav class="navigation-items">
+            <ul class="navigation-list">
+                <li><a class="closeButton" href="index.html">Home</a></li>
+                <li><a class="closeButton" href="aboutme.html">About</a></li>
+            </ul>
+        </nav>
+    </header>
```
Comment (keroloussamy) [inline index.html:36]: - [ ] Kindly add the top short bar section with social media links, logout, and language links aligned to the right. as the original design.

This one
image

File: aboutme.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+        <section class="past-section">
+            <h2 class="past-title">Take a look at the experience!</h2>
+            <div class="red-border"></div>
+            <p class="past-paragraph">Come visit us and join in this life changing adventure!<br> We'll be waiting for you!</p>
+
+            <div class="past-images-div">
+                <img class="pic1" src="./images/background/magic-kingdom-orlando.jpg" alt="picture1">
+                <img class="pic2" src="./images/background/magic-kingdom-orlando2.jpg" alt="picture2">
+            </div>
```
Comment (keroloussamy) [inline aboutme.html]: - [ ] Kindly it's important here to follow the layout, so kindly add some text and color above the two images to match the original design.

Original design
image

Your design
image

File: aboutme.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    <footer class="footer-section">
+        <div class="footer-logo">
+            <img class="footer-logo-img" src="./images/Partners/disney.png" alt="footer-image">
+        </div>
+
+        <div class="footer-texts">
+            <p>2022 Creative Commons Orlando.</p>
+            <p>Some Rights Reserved.</p>
+        </div>
+    </footer>
```
Comment (keroloussamy) [inline aboutme.html:88]: Kindly in the about page
- [ ] The `Partner` section doesn't exist, Kindly remove it.
- [ ] The background for the `footer` section is dark.

Your design
image

Original design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    <footer class="footer-section">
+        <div class="footer-logo">
+            <img class="footer-logo-img" src="./images/Partners/disney.png" alt="footer-image">
+        </div>
+
+        <div class="footer-texts">
+            <p>2022 Creative Commons Orlando.</p>
+            <p>Some Rights Reserved.</p>
+        </div>
+    </footer>
```
Comment (keroloussamy) [inline index.html:109]: - [ ] Kindly add these two sections to the mobile version too on the Home page, I know they are not showing on the original design, but all sections on the desktop commonly appear on the mobile version. it just doesn't appear in the pic.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @VitorGuedesMadeira, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sja-thedude) [reply index.html:36]: - [ ] Great job in adding the top navbar, however, you have only added the social media links, as per the previous reviewer kindly make sure to add logout and language links as per the Figma design.

Screenshot 2022-06-22 at 7 03 45 AM

File: aboutme.html (hunk 2 above)
Comment (sja-thedude) [reply aboutme.html]: Implemented 👍

File: aboutme.html (hunk 3 above)
Comment (sja-thedude) [reply aboutme.html:88]: Implemented 👍

File: index.html (hunk 4 above)
Comment (sja-thedude) [reply index.html:109]: Implemented 👍

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @VitorGuedesMadeira, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Great job on working on your first capstone

 ## Required Changes ♻️
- [ ] _Check the comments under the review._ [here](https://github.com/VitorGuedesMadeira/Capstone/pull/1#discussion_r903261467)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @VitorGuedesMadeira 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commit messages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- :heavy_check_mark: Implemented previous code review requested changes
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (VitorGuedesMadeira) [reply index.html:36]: Thanks for the feedback! Done! ✔️

File: aboutme.html (hunk 2 above)
Comment (VitorGuedesMadeira) [reply aboutme.html]: Now it's perfectly matching the design colors 😄 ✔️

File: aboutme.html (hunk 3 above)
Comment (VitorGuedesMadeira) [reply aboutme.html:88]: Done ✔️

File: index.html (hunk 4 above)
Comment (VitorGuedesMadeira) [reply index.html:109]: Done! ✔️

**Pull request merged**